### PR TITLE
Update sbt installation instructions

### PR DESCRIPTION
**Pull request by schoeberl**
The bintray reference is dead.

### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [ ] Did you delete any extraneous printlns/debugging code?
- [ ] Did you specify the type of improvement?
- [ ] Did you add appropriate documentation in `docs/src`?
- [ ] Did you state the API impact?
- [ ] Did you specify the code generation impact?
- [ ] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you mark the proper milestone (Bug fix: `3.3.x`, [small] API extension: `3.4.x`, API modification or big change: `3.5.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you mark as `Please Merge`?

### CODE REVIEW

**Review by seldridge — APPROVED** [review]
This makes sense to me!  No reason to align with very Ubuntu-specific installation instructions. Good thinking, @schoeberl!

File: SETUP.md [hunk 1]
```diff
@@ -13,14 +13,7 @@ Note that both [PeekPokeTester](https://github.com/freechipsproject/chisel-teste
...
    	
-1.  [Install sbt](http://www.scala-sbt.org/release/docs/Installing-sbt-on-Linux.html),	
-    which isn't available by default in the system package manager:	
-    ```	
-    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list	
-    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 642AC823	
-    sudo apt-get update	
-    sudo apt-get install sbt	
-    ```
+1.  Install sbt according to the instructions from [sbt download](https://www.scala-sbt.org/download.html)
```
Comment (seldridge) [inline SETUP.md:16]:
```suggestion
1.  Install sbt according to the instructions from [sbt download](https://www.scala-sbt.org/download.html).
```

**Review by seldridge — APPROVED** [review]
Still approve, but one grammar nit.

File: SETUP.md (hunk 1 above)
Comment (schoeberl) [reply SETUP.md]: OK,  should be done

**Pull request merged**